src/components/accessories/patientPicker/PatientPicker.tsx

```diff
@@ -252,6 +248,7 @@ const PatientPicker: FC<IProps> = ({
                     errorText={getErrorText("id")}
                     onBlur={formik.handleBlur}
                     type="number"
+                    disabled={searchStatus === "LOADING"}
```

Comment:
Move this control outside the JSX code, inside a variable in order to make this more maintenable.

And then, use the ES6 JS feature in order to pass the disable parameter without the esplicit assignament

Reply:
This is relative to each line `disabled={searchStatus === "LOADING"}`

Reply:
Please can you give me some tips about the **ES6 JS feature ** your talking about?

Reply:
If you have a disabled var you can just put "disabled" without assign a variable explicitly as prop.

Reply:
I tried it but it does'nt work. When we use disabled prop without assigment it will take the default value which is true.

Reply:
I was searching how to disable globally the form or use useFormikContext but it seems not possible

Reply:
So the solution now is just putting *disabled={isLoading}* on every fields